client/lib/analytics/super-props.js

```diff
@@ -14,8 +15,9 @@ const getSuperProps = ( reduxStore ) => ( eventProperties ) => {
 		client: config( 'client_slug' ),
 	};
 
-	const omitSelectedSite =
-		! eventProperties.force_site_id && shouldReportOmitBlogId( eventProperties.path );
+	const path = eventProperties.path ?? getCurrentRoute( state );
```

Comment:
Note that this will fall back to the current route not only if `eventProperties.path` is `undefined`, but also if it is `null`.

Reply:
Any idea where could be a bug causing `path` to be a "random" number between 1 and 103?

It should always be a string. See https://github.com/Automattic/wp-calypso/pull/85821

I investigated [getCurrentRoute](https://github.com/Automattic/wp-calypso/blob/b54b89472ed0df5342e6c153769d08fe3b7f8ed0/client/state/selectors/get-current-route.js#L10) and [ROUTE_SET](https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso%20ROUTE_SET&type=code) without finding anything relevant.